plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/button/style.scss

```diff
@@ -122,3 +122,8 @@
 		border-color: var(--button--color-background);
 	}
 }
+
+.wp-block-button__link.add_to_cart_button:disabled {
+	cursor: not-allowed;
+	opacity: 0.2;
+}
```

Comment:
It would be good to unify this CSS with this one:

https://github.com/woocommerce/woocommerce/blob/71571fe59d6f65e0db91c02adff7e5f5c0db1fce/plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/style.scss#L69-L73

Reply:
I would also increase the `opacity` a bit, to `0.5`, for example. This way the text is easier to read. How does it sound?